graphql/src/cognito.rs

```diff
@@ -0,0 +1,43 @@
+use crate::query::UserInfo;
+use jsonwebtoken::{jwk::JwkSet, Validation};
+
+/// Load the set of keys that can be used to validate Cognito identity tokens
+fn get_jwk_set() -> Result<JwkSet, serde_json::Error> {
+    // this file is pulled from:
+    // https://cognito-idp.<Region>.amazonaws.com/<userPoolId>/.well-known/jwks.json
+    serde_json::from_str::<JwkSet>(include_str!("dailp_user_pool_jwks.json"))
```

Comment:
Should the contents of the JWK Set be in the repo? Id imagine GETting the url listed above would be a safer (and more maintainable) solution.